drivers/hmis/app/graphql/types/hmis_schema/validation_error.rb

```diff
@@ -37,5 +37,10 @@ def type
 
       'UnknownError'
     end
+
+    # Convert 'operatingStartDate' => 'Operating start date'
+    private def readable_attribute
+      object.attribute.to_s.underscore.gsub('_', ' ').capitalize
```

Comment:
@gigxz, I _think_ `object.attribute.to_s.underscore.gsub('_', ' ').capitalize` is equivalent to `object.attribute.to_s.humanize` but it may depend on what the content is.